modules/aws-outposts-load-balancer-clb.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-load-balancer-clb_{context}"]
+= Using the Classic Load Balancer in a mixed remote and cloud-based AWS cluster
+
+(i.e., label non-Outposts machines and make sure only they can attach to a CLB)
```

Comment:
Do we need to provide guidance to CLB? This is something AWS not recommending to be used for years, and AWS recommends to use NLB or ALB instead. I suggest to not add it.

Reply:
Unsure - @JoelSpeed wdyt about this one?

Reply:
Our clusters use CLB by default, as does every AWS cluster running on Kube. Until they change the defaults, I don't see why we would want to omit this. The CCM code is owned by AWS, if they wanted to stop people using CLB, they should stop it from being the default